monai/networks/nets/hovernet.py

```diff
@@ -172,22 +179,24 @@ def __init__(
         dropout_type: Callable = Dropout[Dropout.DROPOUT, 2]
 
         if not drop_first_norm_relu:
-            self.layers.add_module("preact_norm", get_norm_layer(name=norm, spatial_dims=2, channels=in_channels))
```

Comment:
I see that you renamed some of the layers to match the original. I guess this is to allow the pretrained weights to be loaded. I changed the layer names to be somewhat more interpretable whilst allowing for some tweaking of the network. Do we want to change some of them back? If not, then you could just use a dictionary to map the old to the new. 
I'm open to there being a better layer naming convention. The main point, I feel, is that we do it with a purpose and are consistent. (e.g. If we decide to keep the original layer names then we should probably do it for all layers).

Reply:
Thanks @JHancox , this does introduce some breaking changes. Hi @wyli @ericspod , could you please help to make some comment here? The main purpose of changing the name is to simplify the mapping of pretrained weights. If we need to use non-breaking and more interpretable names, I will change them back and add more rules to map the weights.

Reply:
I agree that it is better to be consistent with the naming across MONAI's networks. If we can have a more consistent naming, maybe it can justify the backward incompatibility in the name of layers. @wyli @ericspod @Nic-Ma what do you think?

Reply:
BTW, I think the whole encoder part can be enhanced by making it flexible: not limit to fixed resnet50, and can use more advanced backbones such as seresnext, efficientnet and so on. For example, in the `conic` branch of the referenced repo, the author uses torchvision's resnet, rather than reimplement it (see: https://github.com/vqdang/hover_net/blob/conic/models/hovernet/net_desc.py#L56), it makes the whole network more elegant, and is able to load the torch's pretrained weights directly.
Therefore, in my point of view, considering the possibility of this kind of enhancement, we will introduce many breaking changes.

Reply:
I like the idea of making the encoder interchangeable - makes sense. However, I suspect that @drbeh et al have a deadline to meet so that's probably a decision for him - as it may add complexity. 
As far as the layer naming is concerned, my preference is for whatever is most consistent with the rest of Monai.  Making breaking changes doesn't matter too much at the moment, since we haven't released anything significant yet. The only thing that would break I know of is the new unit test I added to check the kernel consistency with the original model, but I can hold off the PR for this until we have agreed the naming.

Reply:
Hi @JHancox ,

Thanks for your review and discussion here, I agree with you, I feel it's OK to make breaking change for this network.